charts/operator-wandb/charts/parquet/values.yaml

```
@@ -17,6 +17,9 @@ extraCors: []
 
 common: {}
 deployment: {}
+cronJob:
+  enabled: true 
```

```suggestion
  enabled: false
```
lets default this to false.